rclcpp_action/test/test_types.cpp

```diff
@@ -59,3 +60,64 @@ TEST(TestActionTypes, rcl_action_goal_info_to_goal_uuid) {
     EXPECT_EQ(goal_info.goal_id.uuid[i], goal_id[i]);
   }
 }
+
+TEST(TestActionTypes, goal_uuid_hashing_algorithm_collisions) {
+  // Use predefined IDs that have been known to fail with previous hashing algorithms
+  rclcpp_action::GoalUUID g1_1 = {0x0b,0x76,0xbe,0x24,0x5a,0x94,0x42,0xc2,0xfe,0xa2,0x20,0x96,0xa7,0x43,0x4a,0xa7};
```

Comment:
I found in [build report](https://build.ros2.org/job/Rpr__rclcpp__ubuntu_jammy_amd64/1384/#showFailuresLink) 
that line 66,67,69,70,75,76 and 98:
```shell
Lines should be <= 100 characters long
```

Reply:
Thanks @Zard-C.
I removed the test `goal_uuid_hashing_algorithm_collisions`, since the 2nd test already was enough to trigger the problem with the previous hash algorithm.